config/initializers/devise.rb

```diff
@@ -265,7 +265,23 @@
   # ==> OmniAuth
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
-  # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+
+  scheme = Rails.application.config.force_ssl ? "https" : "http"
+  if Rails.application.config.manyfold_features[:oidc]
+    config.omniauth :openid_connect, {
+      name: :openid_connect,
+      issuer: "https://#{ENV["OIDC_PROVIDER_HOST"]}/",
```

Comment:
The issuer might be a subpath on the host (which looks odd, but is entirely in line with the OIDC spec). Hence, I'd rather go with `OIDC_ISSUER` and let users provide the full URI.

This would also rule out nasty issues with triling slahes and such. OIDC generalyl requires the discovery endpoint returning the **exact same string** in the `issuer` field as the string configured here, so admins should be enabled to provide an exact string.

Reply:
Ah, useful to know, I went back and forth on that. Will do :+1: